src/WinUIExSample/MainWindow.xaml

```diff
@@ -20,47 +22,145 @@
+                <Button Click="BringToFront_Click" Content="Bring to front (2 second delay)" />
+                <Button Click="ShowDialog_Click" Content="Show dialog" />
+                <!--<RadioButtons MaxColumns="3" Header="Move Window To: " >
+                <RadioButton Content="TopLeft"  Checked="moveWindowRadioButtons_Checked"/>
+                <RadioButton Content="TopCenter"  Checked="moveWindowRadioButtons_Checked"/>
+                <RadioButton Content="TopRight"  Checked="moveWindowRadioButtons_Checked"/>
+
+                <RadioButton Content="MiddleLeft"  Checked="moveWindowRadioButtons_Checked"/>
+                <RadioButton Content="MiddleCenter"  Checked="moveWindowRadioButtons_Checked"/>
+                <RadioButton Content="MiddleRight"   Checked="moveWindowRadioButtons_Checked"/>
+
+                <RadioButton Content="BottomLeft"  Checked="moveWindowRadioButtons_Checked"/>
+                <RadioButton Content="BottomCenter"  Checked="moveWindowRadioButtons_Checked"/>
+                <RadioButton Content="BottomRight" Checked="moveWindowRadioButtons_Checked"/>
+                </RadioButtons>-->
+                <TextBlock Text="Move Window To:" />
+                <Grid ColumnDefinitions="*,*,*" RowDefinitions="*,*,*">
+                    <Button Click="moveWindowButton_Clicked" Content="TopLeft" />
+                    <Button
+                        Grid.Column="1"
+                        Click="moveWindowButton_Clicked"
+                        Content="TopCenter" />
+                    <Button
+                        Grid.Column="2"
+                        Click="moveWindowButton_Clicked"
+                        Content="TopRight" />
+
+                    <Button
+                        Grid.Row="1"
+                        Click="moveWindowButton_Clicked"
+                        Content="MiddleLeft" />
+                    <Button
+                        Grid.Row="1"
+                        Grid.Column="1"
+                        Click="moveWindowButton_Clicked"
+                        Content="MiddleCenter" />
+                    <Button
+                        Grid.Row="1"
+                        Grid.Column="2"
+                        Click="moveWindowButton_Clicked"
+                        Content="MiddleRight" />
+
+                    <Button
+                        Grid.Row="2"
+                        Click="moveWindowButton_Clicked"
+                        Content="BottomLeft" />
+                    <Button
+                        Grid.Row="2"
+                        Grid.Column="1"
+                        Click="moveWindowButton_Clicked"
+                        Content="BottomCenter" />
+                    <Button
+                        Grid.Row="2"
+                        Grid.Column="2"
+                        Click="moveWindowButton_Clicked"
+                        Content="BottomRight" />
+
+                </Grid>
```

Comment:
Might be cleaner/simpler with a drop down?

Reply:
Not in this case, as the buttons are positioned where they will move the window to.

Reply:
Aaah I totally missed that. Love it!